sdks/go/pkg/beam/core/runtime/graphx/translate.go

```diff
@@ -213,6 +213,7 @@ func (m *marshaller) addScopeTree(s *ScopeTree) string {
 	transform := &pipepb.PTransform{
 		UniqueName:    s.Scope.Name,
 		Subtransforms: subtransforms,
+		EnvironmentId: m.addDefaultEnv(),
```

Comment:
So, it's now the case that Composite Transforms *should* have environments?

Reply:
yes